MqServer 编译报错

### 系统配置信息

RMVL version: 2.x
No OpenCV
OS: Ubuntu 22.04
GCC 11.4.0

### 问题的详细描述

你这编译报错了阿???

```
In file included from /home/frankie/桌面/rmvl/modules/io/src/ipc.cpp:25:
/home/frankie/桌面/rmvl/modules/io/src/ipc.cpp: In constructor ‘rm::MqServer::MqServer(std::string_view)’:
/home/frankie/桌面/rmvl/modules/io/src/ipc.cpp:229:54: error: ‘strerror’ was not declared in this scope; did you mean ‘error’?
  229 |         ERROR_("Failed to create message queue: %s", strerror(errno));
      |                                                      ^~~~~~~~
/home/frankie/桌面/rmvl/modules/core/include/rmvl/core/util.hpp:49:41: note: in definition of macro ‘ERROR_’
   49 |         printf("\033[31m err - \033[0m" __VA_ARGS__); \
      |                                         ^~~~~~~~~~~
/home/frankie/桌面/rmvl/modules/io/src/ipc.cpp: In constructor ‘rm::MqClient::MqClient(std::string_view)’:
/home/frankie/桌面/rmvl/modules/io/src/ipc.cpp:242:58: error: ‘strerror’ was not declared in this scope; did you mean ‘error’?
  242 |         ERROR_("Failed to connect to message queue: %s", strerror(errno));
      |                                                          ^~~~~~~~
/home/frankie/桌面/rmvl/modules/core/include/rmvl/core/util.hpp:49:41: note: in definition of macro ‘ERROR_’
   49 |         printf("\033[31m err - \033[0m" __VA_ARGS__); \
      |                                         ^~~~~~~~~~~
make[2]: *** [modules/io/CMakeFiles/rmvl_io.dir/build.make:90：modules/io/CMakeFiles/rmvl_io.dir/src/ipc.cpp.o] 错误 1
make[2]: *** 正在等待未完成的任务....
make[1]: *** [CMakeFiles/Makefile2:563：modules/io/CMakeFiles/rmvl_io.dir/all] 错误 2
make: *** [Makefile:156：all] 错误 2
```

### 复现步骤

按照官网文档写的

```bash
mkdir build
cd build
cmake ..
make -j14
```

结果报错了

### 工单 Issue 提交清单

- [x] 我报告此代码中的 Bug，我能保证这不是我个人的疑问
- [x] 我在文档、未解决的 Issue 中检查了此问题，但没有找到任何解决方案
- [x] 我尝试过更新到最新的 RMVL 版本，但问题仍然存在
- [x] 我提供了此 Bug 相关的代码和数据文件（视频，图像等）

--- a/modules/io/src/ipc.cpp
+++ b/modules/io/src/ipc.cpp
@@ -22,6 +22,8 @@
 #endif
 #endif
 
+#include <cstring>
+
 #include "rmvl/core/util.hpp"
 #include "rmvl/io/ipc.hpp"
 

